test/common/grpc/async_client_manager_impl_test.cc

```diff
@@ -29,7 +31,9 @@ class RawAsyncClientCacheTest : public testing::Test {
 public:
   RawAsyncClientCacheTest()
       : api_(Api::createApiForTest(time_system_)),
-        dispatcher_(api_->allocateDispatcher("test_thread")), client_cache_(*dispatcher_) {}
+        dispatcher_(api_->allocateDispatcher("test_thread")) {
+    setClientCache(std::chrono::seconds(50));
```

Comment:
This repeats the same pattern as the other test (initializing and then overriding).
RE the need to call initialization on each method:
1. Due to the change of the manager from a non-pointer to a pointer, each test that uses that also needs to be modified, so adding the call to initialize() won't change much.
2. An alternative is to override the Test class (or a shared ancestor), pass the config as a parameter to the c'tor, and then add the new tests to the new Test class. (this should avoid calling initialize on each test).

Reply:
Changed to the initialize() way to keep the test readable.